lib/reports/subscription_changes_after_switch_to_daily_digest_report.rb

```diff
@@ -1,6 +1,13 @@
 class Reports::SubscriptionChangesAfterSwitchToDailyDigestReport
   def self.call
-    updated_subscriptions = Subscription.where(source: 4).where.not(ended_at: nil)
+    experiment_2_list_ids = SubscriberList.where(
+      slug: File.readlines(Rails.root.join("lib/experiment_2_slugs")),
+    )
+
+    updated_subscriptions = Subscription
+      .where(source: 4)
+      .where(subscriber_list_id: experiment_2_list_ids)
```

Comment:
Maybe I've missed something but it looks like this a collection of models and not the ids? Is there a test that executes this code?

Reply:
Oh yes, I forgot to `pluck`. Updated.

I've only tested this manually, since it's only a short-lived report we're running most days.